src/components/MainView.vue

```diff
@@ -1,40 +1,47 @@
 <template>
-	<div class="view-wrapper">
-		<div class="header-wrapper bordered-container">
-		</div>
-		<div class="body-wrapper bordered-container">
-			<div class="left-nav-wrapper bordered-container"></div><div class="body-content-wrapper bordered-container">
-				<recon-console-toolbar
-					v-on:displayFilterDialog_click="displayFilterDialog2"
-					/>
+	<div class="view-resize-wrapper" v-resize="onResize">
+		<div class="view-wrapper">
+			<div class="header-wrapper bordered-container">
 			</div>
+			<div class="body-wrapper bordered-container">
+				<div class="left-nav-wrapper bordered-container"></div><div class="body-content-wrapper bordered-container">
+					<recon-console-toolbar
+						v-on:displayFilterDialog_click="displayFilterDialog2"
+						/>
+				</div>
+			</div>
+			<div class="footer-wrapper bordered-container"></div>
+			<transition name="fade">
+			<keep-alive>
+				<filter-dialog
+					v-bind:filterList="filterList"
+					v-bind:accessLevelList="accessLevelList"
+					v-if="displayFilterDialog"
+					v-on:buttonClick="filterDialogButton_click"
+					/>
+			</keep-alive>
+			</transition>
 		</div>
-		<div class="footer-wrapper bordered-container"></div>
-		<transition name="fade">
-		<keep-alive>
-			<filter-dialog
-				v-bind:filterList="filterList"
-				v-bind:accessLevelList="accessLevelList"
-				v-if="displayFilterDialog"
-				v-on:buttonClick="filterDialogButton_click"
-				/>
-		</keep-alive>
-		</transition>
 	</div>
 </template>
 
 <script>
 import ReconConsoleToolbar from "./ReconConsoleToolbar";
 import FilterDialog from "./FilterDialog";
+//var resize = require("vue-resize-directive");
+import resize from 'vue-resize-directive';
 
 export default {
 	components: {ReconConsoleToolbar, FilterDialog},
+	directives: {
+		resize
+	},
 	mounted: function() {
```

Comment:
It looks like you could remove `mounted` and `beforeDestroy` entirely at this point?

Reply:
Yeah, I left them there as a reminder I guess.